.github/workflows/authors.yml

```diff
@@ -0,0 +1,23 @@
+name: "authors update"
+on:
+  schedule:
+    # 00:00:00 on the first day of every month
+    # https://crontab.guru/#0_0_1_1-12_*
+    - cron: "0 0 * 1-12 *"
```

Comment:
```suggestion
    # https://crontab.guru/#0_0_1_*_*
    - cron: "0 0 * * *"
```

Reply:
Description for the current:

> At 00:00 on day-of-month 1 in every month from January through December.

Description for the change:

> At 00:00 on day-of-month 1 in every month from January through December.

They do the same thing, but `*` is a bit cleaner.

Reply:
Ah, I see the issue - the actual code was wrong but the link was right 😅